docs/reference/run.mdx

```diff
@@ -115,7 +115,49 @@ Runs the `trace_exec` gadget filtering events generated by the pod
 
 ### Filtering in User Space
 
-TODO
+The `--filter` flag allows you to filter events based on specific field values provided by the gadget. This is particularly useful for narrowing down the output to entries that meet certain criteria.
+
+The filter syntax supports the following operations:
+
+```bash
+- `columnName==value`: Matches if the content of `columnName` equals exactly `value`.
```

Comment:
We should talk about field name and not column name because filtering applies to other output format as well, e.g., JSON. Same in [flag description](https://github.com/inspektor-gadget/inspektor-gadget/blob/2f4c687fc695981584497833267e815555df5989/pkg/operators/filter/filter.go#L66-L75).

Reply:
I opened https://github.com/inspektor-gadget/inspektor-gadget/pull/3416 to do that.